app/src/main/java/com/example/fideicomisoapproverring/EscrowApproverActivity.kt

```
@@ -1,71 +1,51 @@
         }
     }
-}
\ No newline at end of file
+
+    // Redirects to LOBSTR Wallet if installed, otherwise, open Play Store
+    private fun connectToLobstrWallet() {
+        try {
+            val lobstrUrl = "lobstr://" // Diagram to open the Lobstr Wallet app
+            val intent = Intent(Intent.ACTION_VIEW, Uri.parse(lobstrUrl))
+            startActivity(intent)
+        } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)